cluster/images/hyperkube/Makefile

```diff
@@ -1,6 +1,9 @@
-# build the hyperkube image.
+# Build the hyperkube image.
+#
+# Usage:
+#   VERSION=v1.1.2 [REGISTRY="gcr.io/google_containers"] make build
 
-VERSION=v1.1.2
+REGISTRY?="gcr.io/google_containers"
```

Comment:
This will out-date really fast! I'm strongly against adding any real value here. If you really insist we can have something like `VERSION=dev` or equivalent.